packages/messenger/src/Messenger.test.ts

```diff
@@ -1264,31 +1264,6 @@ describe('Messenger', () => {
         actions: ['Source:getLength'],
       });
 
-      expect(() => delegatedMessenger.call('Source:getLength', 'test')).toThrow(
-        `Cannot call 'Source:getLength', action not registered.`,
-      );
-    });
-
-    it('unregisters delegated action handlers when action is unregistered', () => {
```

Comment:
Nit: It would be nice to test these cases:

* `throws an error when delegated action is called after handler is unregistered`
* `allows calling delegated action that is unregistered then registered again`

Reply:
I'll add these in a follow-up PR